protos/flyteidl-flink/flink.proto

```diff
@@ -64,6 +64,8 @@ message FlinkJob {
   string image = 8;
   string flinkVersion = 9;
   int32 parallelism = 10 [ (validate.rules).int32.gte = 0 ];
+  string kubernetesClusterName = 11;
+  string region = 12;
```

Comment:
Let's use a more descriptive name like: `kubernetesClusterRegion` for this field as well